enterprise-data-seeding/src/seeders/finance_coa_seeder.py

```
@@ -0,0 +1,290 @@
+        for account_data in STANDARD_COA:
+            try:
+                account_info = {
+                    "account_code": account_data["code"],
+                    "account_name": account_data["name"],
+                    "account_type": account_data["type"],
+                    "balance": 0.0,
+                    "currency": "USD",
+                    "is_active": True,
+                }
+                
+                account, is_new = await self.upsert(Account, account_info, "account_code")
```

Ruff (0.16.2)

[error] Undefined name `Account`

(F821)